src/roles/nuh-deploy/tasks/main.yml

```diff
@@ -81,14 +94,59 @@
 
     - block:
 
+      - name: Create and transfer certs
+        include_role:
+          name: common
+          tasks_from: vsd-generate-transfer-certificates
+        when: not skip_vsd_installed_check
+        vars:
+          certificate_password: "{{ nuh_default_password }}"
+          certificate_username: "proxy"
+          commonName: "proxy"
+          certificate_type: server
+          scp_user: "root"
+          scp_location: /opt/proxy/data/certs
+          additional_parameters:  -d {{ item.external_fqdn }}
+        with_items:
+          - "{{ external_interfaces }}"
+
+      - block:
+        - name: Create notification application user on vsd
+          command: /opt/ejabberd/bin/ejabberdctl register {{ notification_app1.username }} {{ vsd_fqdn }} {{ notification_app1.password }}
```

Comment:
Using password here.  Do we need no_log?

Reply:
done